var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -45,10 +45,15 @@ class Gtkplus(AutotoolsPackage):
     depends_on("pango~X", when='~X')
     depends_on("pango+X", when='+X')
     depends_on('gobject-introspection', when='+X')
+    depends_on('shared-mime-info')
 
     patch('no-demos.patch')
 
     def patch(self):
         # remove disable deprecated flag.
         filter_file(r'CFLAGS="-DGDK_PIXBUF_DISABLE_DEPRECATED $CFLAGS"',
                     '', 'configure', string=True)
+
+    def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
+        spack_env.prepend_path("XDG_DATA_DIRS",
+                               self.prefix.share)
```

Comment:
`spack_env` sets these environment variables at build time. `run_env` sets these environment variables at run time (with module files). Do you only need them set at build time?